### DISCUSSION TITLE
Bulk Deletion of 2FA Tokens/Codes in Ente Auth

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#3911] [Enhancements]
A feature that allows users to delete multiple 2FA tokens/codes at once in the Ente Auth app. Currently, one can only delete tokens one at a time, which can be cumbersome, especially for those with a large number of tokens.

**Comment by nixtoshi** [comment]
Please implement this!

• Button to delete all 2FA seeds
• Button to delete multiple seeds at the same time (less important)

**Comment by kamal-bomoi** [comment]
Please add this! Imported my aegis vault and everything got duplicated, used the delete deduplicated option but now I’ve got 400 entries sitting in trash. No way to clear trash all at once and not feasible to delete all manually. A “clear trash” or “reset account” button would solve this. My only option now is to delete my ente account then wait 7 days for data cleanup then sign up again, quite frustrating considering its the same account i use for ente photos. Oh well
